src/css/hero.css

```
@@ -1,16 +1,178 @@
 /* hero styles */
 .hero-section .container {
+  width: 280px;
+  height: 620px;
   border-radius: 16px;
+  background-image: url(../images/hero/mob_block_1x.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+
+  position: relative;
+}
+
+@media screen and (min-width: 375px) {
+  .hero-section .container {
+    width: 335px;
+    height: 709px;
+  }
+}
+
+@media screen and (max-width: 767.98px) and (min-resolution: 192dpi) {
+  .hero-section .container {
+    background-image: url(../images/hero/mob_block_2x.jpg);
+  }
 }
 
 @media screen and (min-width: 768px) {
   .hero-section .container {
+    width: 704px;
+    height: 857px;
     border-radius: 30px;
+    background-image: url(../images/hero/tab_block_1x.jpg);
+  }
+}
+
+@media screen and (min-width: 768px) and (min-resolution: 192dpi) {
+  .hero-section .container {
+    background-image: url(../images/hero/tab_block_2x.jpg);
   }
 }
 
 @media screen and (min-width: 1280px) {
   .hero-section .container {
     padding: 0 64px;
+    width: 1152px;
+    height: 629px;
+    background-image: url(../images/hero/desktop_block_1x.jpg);
+  }
+}
+
+@media screen and (min-width: 1280px) and (min-resolution: 192dpi) {
+  .hero-section .container {
+    background-image: url(../images/hero/desktop_block_2x.jpg);
+  }
+}
+
+.hero-title {
+  padding-top: 375px;
+  padding-bottom: 30px;
+
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 1.05em;
+  color: var(--secondary-color);
+}
+
+.hero-title-span {
+  color: var(--primary-accent-color);
+}
+
+@media screen and (min-width: 375px) {
+  .hero-title {
+    font-size: 38px;
+    padding-top: 425px;
+    padding-bottom: 36px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .hero-title {
+    padding-top: 260px;
+    width: 488px;
+
+    font-size: 60px;
+    line-height: 1.17em;
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .hero-title {
+    padding-top: 160px;
+    width: 564px;
+  }
+}
+
+.hero-button {
+  border-radius: 12px;
+  border: none;
+
+  width: 240px;
+  height: 52px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 1.71em;
+  color: var(--secondary-color);
+  background-color: var(--button-background);
+}
+
+.hero-button:hover,
+:focus {
```

focus for what? probably you meant 
```suggestion
.hero-button:focus {
```